### DISCUSSION TITLE
Overlap with OneGet (Aka PackageManagement) & Chocolatey

### DISCUSSION THREAD

**Discussion by kilasuit** [microsoft/winget-cli#186] [General]
I feel like we've been here before it seems with Windows Package Management using the OneGet Project (https://docs.microsoft.com/en-us/archive/blogs/packagemanagement/introducing-packagemanagement-in-windows-10) & https://github.com/OneGet & the well used and maintained Chocolatey application that has been in use for over 4 years now & already has tremendous experience with package management on windows and have a decent trajectory going forward.

So considering that there is a product well used out there, what benefit do I as a user of Chocolatey get moving over to winget?
What is the incentive to be invested in this going forward as opposed to Chocolatey?
 
Also as a wider query with this did anyone reach out to any of the contributors/original authors on OneGet like @fearthecowboy @jianyunt, Chocolatey like @ferventcoder or the PowerShell team like @joeyaiello @SteveL-MSFT about this to gain some direction on where they have had pain points with this in the past? If not I'd recommend you look at involving them (if they can find the time) as a steering group if you really want this to be a real long term success.

**Comment by fearthecowboy** [comment]
I'm all behind where they're heading. I've talked briefly to Rich Turner about it, and I'm really quite excited.

I think they're going to be able to succeed where I was not able to. 

Had I not been extremely involved in where I am now, I heavily considered going to work on this. Very very excited

**Comment by kmgallahan** [comment]
> did anyone reach out to any of the contributors/original authors

Per https://devblogs.microsoft.com/commandline/windows-package-manager-preview:

> We have already talked with a few of the well-known package manager teams. Chocolatey has a vibrant community with a massive collection of applications, and a rich history supporting both open source and enterprise customers. Scoop provides a convenient way to allow software to be installed without the UAC popups. Ninite keeps an eye on updates for all the apps it installed. There are many others like AppGet, Npackd and the PowerShell based OneGet package manager-manager.

---

> what benefit do I as a user of Chocolatey get moving over to winget?
> What is the incentive to be invested in this going forward as opposed to Chocolatey?

https://chocolatey.org/pricing

Of course, advanced features would take time to implement.

**Comment by chhantyal** [comment]
> did anyone reach out to any of the contributors/original authors

Looks like they reached out to them https://keivan.io/the-day-appget-died/
